### PR TITLE
Update documentation for configuring facebook login

### PR DESCRIPTION
**Pull request by richvdh**
... and a minor thinko fix in the sample config.

### CODE REVIEW

File: docs/openid.md [hunk 1]
```diff
@@ -390,9 +390,6 @@ oidc_providers:
 
 ### Facebook
 
-Like Github, Facebook provide a custom OAuth2 API rather than an OIDC-compliant
-one so requires a little more configuration.
```
Comment (richvdh) [inline docs/openid.md:394]: it is nearly oidc-compliant now; the difference is spelt out more clearly in the text about discovery at the end.

File: docs/openid.md [hunk 2]
```diff
@@ -412,24 +409,28 @@ Synapse config:
...
-    issuer: "https://facebook.com"
+    issuer: "https://www.facebook.com"
     client_id: "your-client-id" # TO BE FILLED
     client_secret: "your-client-secret" # TO BE FILLED
     scopes: ["openid", "email"]
-    authorization_endpoint: https://facebook.com/dialog/oauth
-    token_endpoint: https://graph.facebook.com/v9.0/oauth/access_token
-    user_profile_method: "userinfo_endpoint"
-    userinfo_endpoint: "https://graph.facebook.com/v9.0/me?fields=id,name,email,picture"
+    authorization_endpoint: "https://facebook.com/dialog/oauth"
```
Comment (DMRobertson) [inline docs/openid.md:416]: Are we quoting values here just for consistency?

**Review by DMRobertson — APPROVED** [review]
I sanity checked that the changes match the changes we've made to matrix.org.

File: docs/openid.md (hunk 2 above)
Comment (richvdh) [reply docs/openid.md:416]: pretty much, yes.

**Pull request merged**